packages/experimental/NativeButton/src/NativeButton.types.ts

```diff
@@ -1,3 +1,4 @@
+import { ColorValue } from '@fluentui-react-native/theme-types';
```

Comment:
My only concern with doing that is we do export ColorValue so removing it might break consumers of the package. If we were to remove the ColorValue type from the package it'd likely be thought of as a breaking change...

Reply:
Looks like we removed it :D